app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -35,12 +37,14 @@ import java.math.BigDecimal
 @Suppress("FunctionNaming", "LongMethod")
 @Composable
 fun CoinItem(
-    coinItem: CoinItemUiModel
+    coinItem: CoinItemUiModel,
+    onMyCoinsItemClick: () -> Unit
```

Comment:
should we set the default value here?

```suggestion
    onMyCoinsItemClick: () -> Unit = {}
```